keras_cv/layers/preprocessing/channel_shuffle.py

```diff
@@ -52,6 +54,11 @@ def __init__(self, groups=3, seed=None, **kwargs):
         self.groups = groups
         self.seed = seed
 
+        tf.random.set_seed(self.seed)
```

Comment:
This is the point in my last comment

Reply:
> @LukeWood cc @bhack
> 
> This line is only set to satisfy the shuffle-net's requirements. By this, it will produce a static shuffled operation.
> 
> ```python
> aug_fn = ChannelShuffle(groups=3, seed=101)
> aug_fn(a, training=True) 
> 
> aug_fn = ChannelShuffle(groups=3, seed=42)
> aug_fn(a, training=True) 
> ```
> 
> But for random operation, `seed=None`. But this may also be a bit in conflict with other KPLs from implementation perspective. Thoughts?

I don't think we should be modifying seed inside of any layer.